conbench/api/auth.py

```diff
@@ -97,8 +97,8 @@ def get(self):
             )
 
         email = google_user["email"]
-        given = google_user["given_name"]
-        family = google_user["family_name"]
+        given = google_user.get("given_name", "")
+        family = google_user.get("family_name", "")
```

Comment:
If an OIDC provider does not yield given_name and family_name as part of the userinfo endpoint then we still want the login flow to succeed. We got their email address which is enough use as user ID. The set of properties returned by the userinfo endpoint by an OIDC provider is _not_ standardized, i.e. for compatibility it's important to expect little.